internal/mtail/mtail.go

```diff
@@ -326,6 +355,15 @@ func (m *Server) Serve() error {
 		}
 		errc <- err
 	}()
+	go func() {
+		ticker := time.NewTicker(10 * time.Second)
+		for {
```

Comment:
This goroutine never gets terminated; add a select case for a quit channel, and close it after m.WaitForShutdown() below.